src/components/avatar-builder/EmojiSelector.js

```diff
@@ -14,25 +15,24 @@ import {
   TouchableOpacity,
   View,
 } from 'react-native';
+import {
+  State,
+  TapGestureHandler,
+  ScrollView,
+} from 'react-native-gesture-handler';
 import LinearGradient from 'react-native-linear-gradient';
-import 'string.fromcodepoint';
-import EmojiTabBarShadow from '../../assets/emojiTabBarShadow.png';
-import { colors, fonts, position } from '../../styles';
-import TabBar from './TabBar';
+import Animated from 'react-native-reanimated';
 import {
   DataProvider,
   LayoutProvider,
   ProgressiveListView,
 } from 'recyclerlistview';
 import StickyContainer from 'recyclerlistview/dist/reactnative/core/StickyContainer';
+import 'string.fromcodepoint';
```

Comment:
Only thing is I dunno what our convention is for alphabetizing this guy "import 'string.fromcodepoint';"

Reply:
think it's fine there for now